mailbox/tools/indexer/src/test/java/org/apache/mailbox/tools/indexer/FullReindexingTaskSerializationTest.java

```diff
@@ -64,6 +70,13 @@ void setUp() {
                 new ReIndexingExecutionFailures.ReIndexingFailure(mailboxId, messageUid),
                 new ReIndexingExecutionFailures.ReIndexingFailure(mailboxId2, messageUid2)),
             ImmutableList.of(mailboxId3));
+
+        recursiveComparisonConfiguration = new RecursiveComparisonConfiguration();
+        recursiveComparisonConfiguration.registerComparatorForType(Comparator.comparingInt(AtomicInteger::get), AtomicInteger.class);
+        recursiveComparisonConfiguration.registerComparatorForType(Comparator.comparingLong(AtomicLong::get), AtomicLong.class);
+        recursiveComparisonConfiguration.registerEqualsForType((o, o2) -> o.get() == o2.get(), AtomicInteger.class);
+        recursiveComparisonConfiguration.registerEqualsForType((o, o2) -> o.get() == o2.get(), AtomicLong.class);
+        recursiveComparisonConfiguration.registerEqualsForType((o, o2) -> o.get() == o2.get(), AtomicBoolean.class);
```

Comment:
Seems it the same most everywhere. For sure we should be able to extract that into a utility class for example?

Reply:
We could